### ISSUE TITLE
How to add 'RG' information to the header of sam/bam

### ISSUE THREAD

**Issue by YingYa** [chhylp123/BitMapperBS#18]
Hi,

How to add 'RG' information to the header of sam/bam?
Could BitMapperBS output CRAM?

Thanks

**Comment by chhylp123** [comment]
It doesn't support 'RG' label for now. If you think it is necessary, I can add it to bitmapperbs.

**Comment by YingYa** [comment]
Yes, the 'RG' information is important for downstream processing (e.g. MarkDuplicates). It would be very helpful that you can add the  'RG' label to bitmapperbs.
